src/app/loginComponents/accounts/external/accounts.component.html

```diff
@@ -14,120 +14,131 @@
+        <span> Show </span>
+      </button>
+    </mat-card>
+  </div>
 
-<div *ngFor="let row of accountsInfo" class="p-3">
-  <mat-card class="mb-2" fxLayout="row">
-    <div fxLayout="column" fxLayoutAlign="center">
-      <img src="../../../../assets/images/account-logos/{{ row.logo }}" alt="{{ row.name }} logo" class="account-logo" />
-    </div>
-    <div style="margin-right: auto">
-      <div>
-        <h3>
-          <mat-chip-list>
-            <mat-chip *ngIf="row.isLinked" color="accent" selected matTooltip="Account is linked">
-              <mat-icon>link</mat-icon>
-            </mat-chip>
-            {{ row.name }} Account&nbsp;
-            <mat-chip [disabled]="true" *ngIf="row.bold" matTooltip="A method of logging into Dockstore">Login Method</mat-chip>
-          </mat-chip-list>
-        </h3>
-      </div>
-      <div>
-        {{ row.message }}
-        <span *ngIf="row.isLinked">If your token has expired, relink your account.</span>
-      </div>
-      <div *ngIf="row.source === TokenSource.ORCID">
-        See <a target="_blank" rel="noopener noreferrer" href="https://info.orcid.org/what-is-orcid/">ORCID</a> and
-        <a target="_blank" rel="noopener noreferrer" [href]="Dockstore.DOCUMENTATION_URL + '/end-user-topics/ORCID.html'">docs</a> for
-        information on what is ORCID.
+  <div *ngFor="let row of accountsInfo" class="p-3" fxFlex="50" fxFlex.lt-md="100">
+    <mat-card class="mb-2 h-100">
+      <div class="h-100" fxLayout="column" fxLayoutAlign="space-between stretch">
+        <div>
+          <div fxLayoutAlign="space-between">
+            <mat-card-header fxLayoutAlign="flex-start" fxLayoutGap="1.5rem">
+              <div mat-card-avatar>
+                <div *ngIf="row.isLinked" matBadge="✓" background="green-color"></div>
+                <img src="../../../../assets/images/account-logos/{{ row.logo }}" alt="{{ row.name }} logo" class="account-logo" />
+              </div>
+              <mat-card-title class="title">{{ row.name }} Account&nbsp;</mat-card-title>
+              <mat-card-subtitle class="accent-1-dark" *ngIf="row.isLinked; else placeholder">
+                <strong>Username: {{ row.source | getTokenUsername: tokens }} </strong>
+                <span *ngIf="row.name === 'ORCID'">
+                  <strong> ID: </strong>
+                  <a href="https://orcid.org/{{ orcidId$ | async }}"> {{ orcidId$ | async }}</a>
+                </span>
+              </mat-card-subtitle>
+              <ng-template #placeholder><br /></ng-template>
+            </mat-card-header>
+            <mat-chip-list>
+              <mat-chip class="gray-label-tag" *ngIf="row.control" matTooltip="TBD"> Source Control </mat-chip>
+              <mat-chip class="gray-label-tag" *ngIf="row.bold" matTooltip="A method of logging into Dockstore"> Login Method </mat-chip>
+              <mat-chip class="gray-label-tag" *ngIf="row.docker" matTooltip="TBD"> Docker Repository </mat-chip>
```

Comment:
Maybe 'Link Docker repository accounts to help Dockstore find information on your Docker containers`